test/tree_sitter/query_test.rb

```diff
@@ -152,6 +152,25 @@ def mul(a, b)
   end
 end
 
+describe 'captures and matches' do
+  it 'should be enumerbale' do
+    src = <<~MATH
+      1 + x * 3
+    MATH
+    math = TreeSitter.lang('math')
+    parser = TreeSitter::Parser.new
+    parser.language = math
+    tree = parser.parse_string(nil, src)
+    query_string = '(sum) @sum'
+
+    query = TreeSitter::Query.new(math, query_string)
+    matches = TreeSitter::QueryCursor.new.matches(query, tree.root_node, src)
+    captures = TreeSitter::QueryCursor.new.captures(query, tree.root_node, src)
+    matches.class.ancestors.include?(Enumerable)
+    captures.class.ancestors.include?(Enumerable)
```

Comment:
While this is technically unbeatably accurate, this is also quite disappointing in the content, and in the form.

In the content, I would at least check a few (interesting) calls (which you did below, granted).

In the form, well, this is not a test at all :)  You mean
```ruby
_(matches.class.ancestors).must_be :include?, Enumerable
```

Reply:
For the form, yes, I screwed while removing the specific function calls you mention on content :) I thought it was too much … Will bring them back.